src/app/cart/templates/cart.html

```diff
@@ -32,7 +32,7 @@ <h5 ng-repeat="promotion in cart.promotions" class="text-danger">
                         <span class="pull-right">- {{promotion.Amount | currency}}</span></h5>
                 </div>
                 <div class="panel-footer">
-                    <button ng-if="cart.lineItems.Items.length > 0" type="button" ui-sref="checkout.shipping" class="btn btn-block btn-primary" ng-disabled="OrderLineItemForm.$invalid">Proceed to Checkout</button>
+                    <button ng-if="cart.lineItems.Items.length > 0" type="button" ng-click="cart.proceedToCheckout()" class="btn btn-block btn-primary" ng-disabled="OrderLineItemForm.$invalid">Proceed to Checkout</button>
                 </div>
```

Comment:
I thought about removing it, but while developing it took me a second to figure out why my click wasn't updating and that's because we have TWO proceed to checkout buttons (one only displays on mobile views).

I think leaving this as a function will help future developers alter the proceedToCheckout behavior in a way that doesn't require them to know anything about the template.